### PR TITLE
Teleop bindings

### PR DESCRIPTION
**Pull request by Lewis-Seiden**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -124,32 +135,101 @@ public void robotInit() {
...
     intake.setDefaultCommand(intake.runVoltageCmd(10.0));
-    shooter.setDefaultCommand(shooter.runStateCmd(Rotation2d.fromDegrees(0.0), 0.0, 0.0));
+    shooter.setDefaultCommand(
+        shooter.runStateCmd(
+            () -> Rotation2d.fromDegrees(0.0), () -> flywheelIdleSpeed, () -> flywheelIdleSpeed));
     reactionBarRelease.setDefaultCommand(
         reactionBarRelease.setRotationCmd(Rotation2d.fromDegrees(0.0)));
 
-    // Controller bindings here
+    controller.setDefaultCommand(controller.rumbleCmd(0.0, 0.0));
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/Robot.java:152]: duplicate comments?

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -124,32 +135,101 @@ public void robotInit() {
...
+
+    // ---- Controller bindings here ----
+    controller.leftTrigger().whileTrue(intake.runVoltageCmd(10.0));
     controller
         .rightTrigger()
-        .whileTrue(shooter.runStateCmd(Rotation2d.fromDegrees(45.0), 50.0, 40.0));
-    controller.start().onTrue(Commands.runOnce(() -> swerve.setYaw(Rotation2d.fromDegrees(0))));
-
-    // Test binding for autoaim
+        .and(() -> currentTarget == Target.SPEAKER)
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/Robot.java:175]: maybe contrary to the formatting, i might put these on the same line, so it reads better like:
```suggestion
.rightTrigger().and(() -> currentTarget == Target.SPEAKER)
```

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -124,32 +135,101 @@ public void robotInit() {
...
+                Commands.parallel(
+                    elevator.setExtensionCmd(() -> ElevatorSubsystem.TRAP_EXTENSION_METERS),
+                    Commands.waitUntil(
+                            () ->
+                                elevator.getExtensionMeters()
+                                    > 0.95 * ElevatorSubsystem.TRAP_EXTENSION_METERS)
+                        .andThen(carriage.runVoltageCmd(-3.0)))));
+
+    // Prep climb
+    operator
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/Robot.java:221]: try to be consistent with line breaks (small things)

**Review by Awesomeplayer165 — APPROVED** [review]
Looks good, just small formatting things.

**Comment by Lewis-Seiden** [comment]
those lines dont appear on my machine

**Pull request merged**